cpp/include/Ice/LocalExceptions.h

```diff
@@ -683,25 +683,23 @@ namespace Ice
         [[nodiscard]] const char* ice_id() const noexcept final;
     };
 
-    /// The exception that is thrown when you attempt to find or deregister something that is not registered with the
-    /// Ice runtime or Ice locator.
+    /// An attempt was made to find or deregister something that is not registered with the Ice run time or Ice
```

Comment:
The latest style for runtime is runtime without space or dash.

Reply:
I would actually simplify here and write:
```
An attempt was made to find or deregister something that is not registered with Ice.
```

Where "with Ice" can mean an Ice communicator, an Ice locator implementation, an object adapter, etc. Various Ice-provided containers.